src/dmd/aggregate.h

```diff
@@ -161,7 +162,7 @@ struct StructFlags
 class StructDeclaration : public AggregateDeclaration
 {
 public:
-    int zeroInit;               // !=0 if initialize with 0 fill
+    int zeroInit;               // -1 if still unknown, 1 if initialize with 0 fill, 0 otherwise
```

Comment:
Can we please start using enums for things like this?

Reply:
I didn't yet do it to reduce the diff, but now added an enum.